src/styles/components/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  &__star {
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--1 > :nth-child(-n + 1),
+  &--2 > :nth-child(-n + 2),
+  &--3 > :nth-child(-n + 3),
+  &--4 > :nth-child(-n + 4),
+  &--5 > :nth-child(-n + 5) {
```

Comment:
you can use loop with [@for](https://sass-lang.com/documentation/at-rules/control/for/)